src/Applications/GEOSdas_App/fvsetup

```diff
@@ -9781,12 +9795,13 @@ sub init_agcm_rc {
     $envvars{"coupled"} = $coupled;
     set_AGCM_envvars(%envvars);
 
-    $flags{"gocart_tracers"} = $gocart_tracers;
     $flags{"carma_tracers"}  = $carma_tracers;
+    $flags{"gocart_tracers"} = $gocart_tracers;
     $flags{"iau"}            = $doiau;
-    $flags{"pcp_forced"}     = $pcp_forced;
-    $flags{"lsmodel_flag"}   = $lsmodel_flag;
     $flags{"ldas_flag"}      = $ldas_flag;
+    $flags{"lsmodel_flag"}   = $lsmodel_flag;
+    $flags{"pcp_forced"}     = $pcp_forced;
+
```

Comment:
Same as above ... Are we intentionally changing the order of the two lines here (relative to that in develop)?  Unless there's a reason for the revised order, we shouldn't modify develop

Reply:
the order of the two lines are not changed in the code, not sure why it is indicated here.

Reply:
the order of the lines does seem changed, see comment above

Reply:
same as above